File: src/main/java/io/mfedirko/learning/Lesson.java
package io.mfedirko.learning;

import lombok.Builder;
import lombok.Getter;
import lombok.Value;
import lombok.extern.jackson.Jacksonized;

import java.time.LocalDateTime;

@Getter
@Builder
@Value
@Jacksonized
public class Lesson {
    LocalDateTime creationTimestamp;
    long creationTimestampMillis;
    String author;
    String category;
    String title;
    String description;
    String parsedDescription;
}

File: src/main/kotlin/io/mfedirko/learning/Lesson.kt
package io.mfedirko.learning

import java.time.LocalDateTime
import kotlin.properties.Delegates

class Lesson {
    var creationTimestamp: LocalDateTime by Delegates.notNull()
    var id: Any by Delegates.notNull()
    var author: String by Delegates.notNull()
    var category: String by Delegates.notNull()
    var title: String by Delegates.notNull()
    var description: String by Delegates.notNull()
    var parsedDescription: String by Delegates.notNull()
}